<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingDialog</class>
 <widget class="QDialog" name="BookingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flight Booking</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Flight Booking</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="departureLabel">
       <property name="text">
        <string>Departure:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="departureComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="destinationLabel">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="destinationComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Passenger Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="cnicLabel">
       <property name="text">
        <string>CNIC:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="cnicEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="searchButton">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Search Flights</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="flightsTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Flight No.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bookButton">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Book Flight</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
